lib/jekyll-admin/apiable.rb

```diff
@@ -38,20 +37,26 @@ def to_api(include_content: false, include_documents: false)
       output.delete("output")
 
       # By default, calling to_liquid on a collection will return a docs
-      # array with each rendered document, which we don't want. Instead
-      # include our own documents array, without the content
+      # array with each rendered document, which we don't want.
       if is_a?(Jekyll::Collection)
         output.delete("docs")
-        if include_documents
-          output["documents"] = docs.sort_by(&:date).reverse.map(&:to_api)
-        end
+      end
+
+      if is_a?(Jekyll::Document)
+        output["dir"] = File.join("/", relative_path_from_source.dirname.to_s)
+        output["name"] = relative_path_from_source.basename
```

Comment:
For non-Documents, this should already be calculated as `name`. For documents, this should be already calculated as `basename_without_ext`.